test/integration/convert_test.go

```diff
@@ -0,0 +1,67 @@
+package integration
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/anchore/syft/cmd/syft/cli/convert"
+	"github.com/anchore/syft/internal/config"
+	"github.com/anchore/syft/internal/formats/syftjson"
+	"github.com/anchore/syft/internal/formats/table"
+	"github.com/anchore/syft/syft"
+	"github.com/anchore/syft/syft/source"
+	"github.com/stretchr/testify/require"
+)
+
+// TestConvertCmd tests if the converted SBOM is a valid document according
+// to spec.
+// FIXME: This test can, but currently does not, check the converted SBOM content. It
```

Comment:
nit: I'd say "TODO" instead of "FIXME", so that we don't need to search for an additional string as we find TODO items